easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-intel-2016b.eb

```diff
@@ -13,9 +13,11 @@ description = """OpenPGM is an open source implementation of the Pragmatic Gener
 toolchain = {'name': 'intel', 'version': '2016b'}
 toolchainopts = {'pic': True}
 
-source_urls = [GOOGLECODE_SOURCE]
+source_urls = ['https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/openpgm/']
 sources = ['libpgm-%(version)s.tar.gz']
 
+checksums = ['d7673e9ff6cc33cf42fe9fb9a7bfbffa']
```

Comment:
Comment: that checksum is what I got on the local clusters after downloading from the google archives.  It is not a published checksum from google.